### PR TITLE
Add telemetry API

### PR DESCRIPTION
**Pull request by kirugan**

### CODE REVIEW

File: cmd/juno/juno.go [hunk 1]
```diff
@@ -202,6 +203,7 @@ func main() {
 	})
 
 	if err := cmd.ExecuteContext(ctx); err != nil {
+		fmt.Fprintf(os.Stderr, "error: %v\n", err)
```
Comment (IronGauntlets) [inline cmd/juno/juno.go:206]: I think this is redundant because cobra prints the error

File: telemetry/service.go [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+	for {
+		select {
+		case <-ctx.Done():
+			return nil
+		case <-ticker.C:
+			err := s.sendTelemetry(ctx)
+			if err != nil {
+				s.log.Errorw("Failed to send telemetry", "err", err)
+			}
```
Comment (IronGauntlets) [inline telemetry/service.go:44]: Do you think we should sleep for a bit before the next iteration of the loop?

Comment (kirugan) [reply telemetry/service.go:44]: I thought that ticker should do that, though if sending takes 5 sec next request will start immediately. We can update ticker once request is complete, wdyt?

File: cmd/juno/juno.go (hunk 1 above)
Comment (kirugan) [reply cmd/juno/juno.go:206]: Thanks, forgot about it.

**Pull request closed without merging**